testing/logging/test_fixture.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+import sys
+import logging
+
+
+logger = logging.getLogger(__name__)
+sublogger = logging.getLogger(__name__+'.baz')
+
+u = (lambda x: x.decode('utf-8')) if sys.version_info < (3,) else (lambda x: x)
```

Comment:
I don't think you need this, you can just use plain unicode literals (use `u'bū'` instead of `u('bū')`). This code was needed for Python <3.3 which reintroduced the `u` prefix for unicode literals.

Reply:
Btw just to be clear: I know you just ported the code and didn't wrote that bit. My suggestions are small improvements before merging the code. 😁